python/cac_tripplanner/cac_tripplanner/settings.py

```diff
@@ -0,0 +1,96 @@
+"""
+Django settings for cac_tripplanner project.
```

Comment:
I'm not sure I think this is a good idea; I personally find the apps where we template the whole settings.py file easier to work with than the ones where we split out the secrets, because it minimizes the amount of sleuthing you need to do to figure out what the value of a given setting is for a running application. That said, this type of thing rarely causes serious issues with a project, and this is a slightly different take on it than we've done before, so I'm willing to experiment.